graphstorm-processing/graphstorm_processing/data_transformations/dist_transformations/dist_noop_transformation.py

```diff
@@ -0,0 +1,88 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License").
+You may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from typing import List, Optional
+
+from pyspark.sql import DataFrame
+from pyspark.sql import functions as F
+from pyspark.sql.types import ArrayType, FloatType, NumericType
+
+from graphstorm_processing.constants import SPECIAL_CHARACTERS
+from .base_dist_transformation import DistributedTransformation
+
+
+class NoopTransformation(DistributedTransformation):
+    """
+    A no-op transformation that parses data as floats or lists of floats
+    and forwards the result withouth any processing.
+    """
+
+    def __init__(self, cols: List[str], separator: Optional[str] = None) -> None:
+        super().__init__(cols)
+        # TODO: Support multiple cols?
+
+        self.separator = separator
```

Comment:
I wanted to handle vector CSV columns as well, so that a csv such as:

```
col1,col2,no-op-col
val1,val2,1|2|3
val3,val4,4|5|6
```

can be parsed using `|` as a separator. Do you think we should remove that?